tools/scan_yaml_for_risky_text.py.py

```diff
@@ -0,0 +1,19 @@
+"""Check for security issues in workflows files."""
+
+import sys
+from pathlib import Path
+
+# according to this article: https://nathandavison.com/blog/github-actions-and-the-threat-of-malicious-pull-requests
+# we should avoid using `pull_request_target` for security reasons
+risky_text = "pull_request_target"
+
+
+def find_risky_files(path: str):
+    """Searching for risky text in yml files for given path."""
+    return {str(file) for file in Path(path).rglob("*.yml") if f"{risky_text}" in file.read_text()}
```

Comment:
```suggestion
    return {file for file in Path(path).rglob("*.yml") if risky_text in file.read_text()}
```
since the `str` conversion will be done already in the `sys.exit`.

Reply:
Thanks, working!)